src/components/NcRichText/NcRichText.vue

```diff
@@ -366,6 +438,27 @@ export default {
 						)
 
 						if (!tag.startsWith('#')) {
+							if (this.useExtendedMarkdown) {
+								if (tag === 'li' && Array.isArray(children)
+									&& children[0].tag === 'input'
+									&& children[0].data.attrs.type === 'checkbox') {
+									const [inputNode, , label] = children
+									const id = 'markdown-input-' + GenRandomId(5)
+									const inputComponent = h(NcCheckboxRadioSwitch, {
+										attrs: {
+											...inputNode.data.attrs,
+											id,
+											disabled: !this.interactive,
+										},
+										on: {
+											'update:checked': (value) => {
+												this.$emit('interacted', { id, label, value })
```

Comment:
Events are usually named as action name, e.g. `click`, not `clicked`.

```suggestion
												this.$emit('interact', { id, label, value })
```

Reply:
But according to the payload, this event (or at least with this payload) is supposed to be only `check` state in todo change.

So I'd call it explicitly `check` or `todo-check` or something like that.

Or, if we expect more interactions in the future, kind of 

```js
this.$emit('interact', { for: 'todo', id, label, value })
this.$emit('interact:todo', { id, label, value })
```

Reply:
> Sounds like a good idea, then? `this.$emit('interact', event)` for other elements with native events, if they ever appear `this.$emit('interact:todo', { id, label, value })` for checkboxes

I meant `interact` for any interaction in case we need to just handle all interactions, and `interaction:<name>` for a specific kind of interaction